invenio_pidstore/provider.py

```diff
@@ -48,6 +48,14 @@ class PIDStatus:
     registered.
     """
 
+    @classmethod
+    def list_in_order(cls):
```

Comment:
Yes, but are we going to re-define the labels for UI? Admin interface can benefit from it.